.gitignore

```diff
@@ -1,3 +1,7 @@
+
+# this is a test
+
+
```

Comment:
hey please remember-- Access control enforces policy such that users cannot act outside of their intended permissions.

📄 [OWASP](https://owasp.org/Top10/A01_2021-Broken_Access_Control/)

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix&nbsp; | &nbsp;**Tags:** Not secure

Reply:
now it looks good

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :ok_hand: This code looks good

Reply:
now it's awesome and elegant

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome&nbsp; | &nbsp;**Tags:** Elegant

Reply:
this refactoring is awesome

__
[![sema-logo](https://app.semasoftware.com/img/sema-tray-logo.svg)](https://semasoftware.com/gh) &nbsp;**Summary:** :trophy: This code is awesome